PhysicsTools/Heppy/python/physicsutils/JetReCalibrator.py

```diff
@@ -109,12 +131,19 @@ def getCorrection(self,jet,rho,delta=0,corrector=None,uncertainty="Total"):
         corrector.setJetA(jet.jetArea())
         corrector.setRho(rho)
         corr = corrector.getCorrection()
-        if delta != 0:
-            JetUncertainty = self.factorizedUncertainties[uncertainty]
-            if not JetUncertainty:
-                raise RuntimeError("Jet energy scale uncertainty shifts requested, but not available")
-            JetUncertainty.setJetEta(jet.eta())
-            JetUncertainty.setJetPt(corr * jet.pt() * jet.rawFactor())
+
+        # if delta != 0:
+        # Short circuit the condition so we always attach the uncertainty to the jet
+        # (Why was this only done for delta != 0?)
+        if True:
```

Comment:
I think it was left in case some people want to reimplement the "if delta!=0" but I agree it can be removed. @gkasieczka @jpata I do not remember who did that but I'll do the fix